src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,41 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by '}
+          {user && (
+            <UserInfo user={user} />
+          )}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
+
+      {comments.length !== 0
+        ? <CommentList comments={comments} />
+        : <b data-cy="NoCommentsMessage">No comments yet</b>}
```

Comment:
```suggestion
        : <b data-cy="NoCommentsMessage">No comments yet</b>
      }
```

Reply:
fixed

Reply:
not fixed :)

<img width="551" alt="image" src="https://user-images.githubusercontent.com/86183007/229353862-e4d13723-7cfa-4498-a7d6-caa476f00a47.png">

Reply:
![image](https://user-images.githubusercontent.com/120763734/229359076-697b1fc7-63c3-4150-ba32-5a525b78e31f.png)

thank you
with this coding style
readability and maintainability are definitely better